app/_hub/kong-inc/request-termination/index.md

```diff
@@ -69,16 +73,31 @@ params:
       default: "`application/json; charset=utf-8`"
       datatype: string
       description: Content type of the raw response configured with `config.body`.
+    - name: trigger
+      required: false
+      default:
+      description: When not set, the plugin always activates. When set to a string, the plugin will activate exclusively on requests containing either a header or a query parameter named like said string.
```

Comment:
```suggestion
      description: When not set, the plugin always activates. When set to a string, the plugin will activate exclusively on requests containing either a header or a query parameter that includes the string.
```

Reply:
I think "is named like" is better than "includes" here. Explanation:

A header (or a query parameter) has two parts: the name and the value. For example in `My-Header: 14` the string `My-Header` is the name and `14` is the value.

When I say the header is "named like said string" I mean that its name must be literally the same. For example, if `trigger` is set to `Activate-Debug` then this header: `Activate-Debug: true` will be named the same, because its name is the same. The value can be anything, even the empty value 

A header that **contains** the string, on the other hand, could be be `My-Activate-Debug: true`, on which the string of the name contains `Activate-Debug` as a substring) or `My-Something-Other: Activate-Debug`, where it is on its value.

If there is another way to say what I meant in English, I don't know it. Perhaps I should add an example?

Reply:
how's that?

Reply:
Good enough! Accepted the change